adit/dicom_web/tests/acceptance/test_wadors.py

```diff
@@ -37,6 +41,58 @@ def test_retrieve_study(channels_live_server: ChannelsLiveServer):
     )
 
 
+@pytest.mark.acceptance
+@pytest.mark.order("last")
+@pytest.mark.django_db(transaction=True)
+def test_retrieve_study_with_pseudonym(channels_live_server: ChannelsLiveServer):
+    setup_dimse_orthancs()
+
+    _, group, token = create_user_with_dicom_web_group_and_token()
+    server = DicomServer.objects.get(ae_title="ORTHANC1")
+    grant_access(group, server, source=True)
+    adit_client = AditClient(server_url=channels_live_server.url, auth_token=token)
+
+    metadata = load_sample_dicoms_metadata("1001")
+    study_uid: str = metadata.iloc[0]["StudyInstanceUID"]
+    test_pseudonym = "TestPseudonym"
+
+    results = adit_client.retrieve_study(server.ae_title, study_uid, test_pseudonym)
+    for result in results:
+        assert isinstance(result, pydicom.Dataset)
+        assert result.PatientID == test_pseudonym
+        assert result.PatientName == test_pseudonym
```

Comment:
Why call it `test_pseudonym` and not just `pseudonym`? Let's keep variables simple when they are nothing special (like mocks or so).